### PR TITLE
LPS-76030

### PR DESCRIPTION
**Pull request by noahsherrillwork**
Changes made per pei-jung#234.

/cc @drewbrokke, @willnewbury, @JorgeFerrer

### CODE REVIEW

**Comment by noahsherrillwork** [comment]
ci:test

File: modules/apps/collaboration/bookmarks/bookmarks-uad/src/main/java/com/liferay/bookmarks/uad/anonymizer/BookmarksEntryUADEntityAnonymizer.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	@Reference
+	private UADAnonymizerHelper _uadAnonymizerHelper;
+
+	@Reference(
+		target = "(model.class.name=" + BookmarksUADConstants.BOOKMARKS_ENTRY + ")"
+	)
+	private UADEntityAggregator _uadEntityAggregator;
+
+	@Reference
+	private UserLocalService _userLocalService;
```
Comment (drewbrokke) [inline modules/apps/collaboration/bookmarks/bookmarks-uad/src/main/java/com/liferay/bookmarks/uad/anonymizer/BookmarksEntryUADEntityAnonymizer.java]: Is this used at all?

Comment (noahsherrillwork) [reply modules/apps/collaboration/bookmarks/bookmarks-uad/src/main/java/com/liferay/bookmarks/uad/anonymizer/BookmarksEntryUADEntityAnonymizer.java]: No, it isn't; it's a remnant of a previous version that was never removed.

**Comment by drewbrokke** [comment]
@noahsherrillwork LGTM 👍 except for the one leftover field.

**Comment by pei-jung** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/54453. Thanks!

**Pull request closed without merging**